### ISSUE TITLE
Sum HP gain and loss

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#268]
```
reported here:
http://andorstrail.com/viewtopic.php?f=4&t=1112

Several HP changing effects have the same visual effect like a "+1" and a "-1" 
floating above the player. They sometimes overlap.

There should be the best to show the sum of the effects (from 1x poison, 1x 
RoLS and 2x Regeneration skill it should result in a "+2")
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 3 Nov 2011 at 6:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `sdeva...@gmail.com` on 3 Nov 2011 at 6:40
- Changed state: **Accepted**
- Added labels: **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
This was actually implemented a while ago (I forgot the revision number though).

Thanks to ScorpionZ for the idea!
```

Original comment by `oskar.wi...@gmail.com` on 12 Mar 2012 at 9:40
- Changed state: **Ready**
- Added labels: **Type-Enhancement**, **Milestone-0.6.11**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
